src/pages/careers/index.jsx

```diff
@@ -0,0 +1,67 @@
+import { withTranslation } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import AvailableJobs from "@/components/AvailableJobs";
+import CareerPhilosophy from "@/components/CareerPhilosophy";
+import getDocument from "@/firebase/getData";
+import PageIntro from "@/components/PageIntro";
+import Button from "@/components/ui/Button";
+import Router from "next/router";
+function career({ t, careers }) {
+    const handleClick = () => {
+        const router = require("next/router").default;
+        router.push({
+            pathname: "/careers/create",
+        });
+    };
+    return (
+        <div>
+            <div className=' lg:ms-28 mt-10 ms-10 '>
+                <PageIntro
+                    title={t("careerAtHealing")}
+                    subtitle={t("bringHopeThroughTherapy")}
+                    t={t}
```

Comment:
- Page intro component does not accept the last `t` prop, so please remove it and send `title` and `subtitle` as localized strings.

Reply:
I didn't write this part of code ,I edit it  ,Done